js/piwik.js

```diff
@@ -592,6 +592,19 @@ if (typeof window.Matomo !== 'object') {
 
             return matches ? matches[1] : url;
         }
+        function queryStringify(data, filterFn, throwErrorIfNotFilterFn) {
+            var queryString = '', k, filter = filterFn || function(value) { return value; }, isOk;
```

Comment:
Seems the `isOk` variable isn't used anymore.

```suggestion
            var queryString = '', k, filter = filterFn || function(value) { return value; };
```